Makefile

```diff
@@ -0,0 +1,24 @@
+# Image can be overidden with an env var.
+DOCKER_IMAGE?=microscaling/microscaling
+
+# Version tag from the code.
+VERSION=`cat VERSION`
```

Comment:
My only concern here is whether we want to tag the Docker file as version N, if it's actually version N + some commits?